tesseract_state_solver/src/ofkt_state_solver.cpp

```diff
@@ -879,7 +879,7 @@ bool OFKTStateSolver::initHelper(const tesseract_scene_graph::SceneGraph& scene_
 void OFKTStateSolver::moveLinkHelper(std::vector<JointLimits::ConstPtr>& new_joint_limits, const Joint& joint)
 {
   auto* old_node = link_map_[joint.child_link_name];
-  const std::string& old_joint_name = old_node->getJointName();
+  const std::string old_joint_name = old_node->getJointName();
```

Comment:
Was this causing an issue?

Reply:
`old_joint_name`  is being used after `old_node` is modified/deleted. I am not sure how it was working before.

Reply:
Oh, I see the issue right after this the node is removed which invalidates the reference.